src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -37,6 +37,7 @@ class ReviewPage extends React.Component {
           formConfig={formConfig}
           pageList={pageList}
           path={path}
+          renderErrorMessage
```

Comment:
I can open this up in another PR if necessary, GenericError wont hit the customError message without this hardcoded boolean